cmake/BipedalLocomotionFrameworkFindDependencies.cmake

```diff
@@ -147,3 +147,7 @@ framework_dependent_option(FRAMEWORK_COMPILE_Planners
 framework_dependent_option(FRAMEWORK_COMPILE_ContactModels
   "Compile ContactModels library?" ON
   FRAMEWORK_HAS_Eigen3 OFF)
+
+framework_dependent_option(FRAMEWORK_COMPILE_System
+  "Compile Estimators library?" ON
```

Comment:
Copy paste? :grin:

Reply:
Fixed in 743c4b383cf7b41f7e5c1acb7270261fac1e0f16